h2o-py/h2o/schemas/error.py

```diff
@@ -21,6 +21,7 @@ def __init__(self):
         super(H2OErrorV3, self).__init__()
         self.endpoint = None
         self.payload = None
+        self.show_stacktrace = True
```

Comment:
Do we know what other subclasses are affected? Does this lead to too verbose output in some cases?

Reply:
I disable the stacktrace for client errors, as the message should be enough for those.
I consider that all other errors coming from the backend are unexpected and deserve to display the stacktrace.

Reply:
Thanks for explaining